src/SceneSwitcher.svelte

```diff
@@ -18,15 +18,11 @@
   $: window.localStorage.setItem('sceneIcons', JSON.stringify(sceneIcons))
 
   onMount(async function () {
-    let data = await sendCommand('GetSceneList')
+    const data = await sendCommand('GetSceneList')
+    console.log('GetSceneList', data)
     programScene = data.currentProgramSceneName || ''
+    previewScene = data.currentPreviewSceneName
     scenes = data.scenes
-    console.log('GetSceneList', data)
-    data = await sendCommand('GetStudioModeEnabled')
```

Comment:
If we remove this part, OBS-web will malfunction when connecting to a studio-mode enabled OBS. It think it's in program mode so every scene switch will switch the program instead of the preview.